keras_cv/layers/preprocessing/random_height.py

```diff
@@ -0,0 +1,129 @@
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing.base_image_augmentation_layer import (
+    BaseImageAugmentationLayer,
+)
+from keras_cv.utils import preprocessing
+
+BILINEAR = "bilinear"
+NEAREST = "nearest"
+BICUBIC = "bicubic"
+AREA = "area"
+LANCZOS3 = "lanczos3"
+GAUSSIAN = "gaussian"
+MITCHELLCUBIC = "mitchellcubic"
```

Comment:
tf.image.ResizeMethod is already string based, prefer using those

Reply:
Will do!

Reply:
Lets use the utility from my PR for Resizing.  `keras_cv.utils.get_interpolation()`.

Reply:
Alright, will do that instead!